src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,101 @@
+package homework_2.random_chars_table;
+
+
+import java.util.Scanner;
+
+public class RandomCharsTable {
+    protected static final int MIN_ASCII_CODE = 65;
+    protected static final int MAX_ASCII_CODE = 90;
+
+    public void run(){
+
+        System.out.println("Enter the dimensions of matrix and required strategy : two integers and required strategy - even|odd, delimiter - space character))");
+
+        Scanner scanner = new Scanner(System.in);
+        String inString = scanner.nextLine();
+        String tempStr = "";
+        char c;
+        int cols =0; int rows =0;
+        String strategy = "";
+        int paramCount=0;
+                // parse of input procedure
```

Comment:
If you need a comment, means you can move it to a separate method

Reply:
Done (another method was created).